ops/model.py

```diff
@@ -50,7 +51,7 @@ class Model:
     """
 
     def __init__(self, meta: 'ops.charm.CharmMeta', backend: '_ModelBackend'):
-        self._cache = _ModelCache(backend)
+        self._cache = _ModelCache(backend, meta)
```

Comment:
No reason, except "it seemed better to append the new parameter rather than insert". But this is a purely internal class so makes no difference -- fixed.